mcs/class/corlib/corefx/AesCcm.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Security.Cryptography
+{
+    public sealed partial class AesCcm : System.IDisposable
+    {
+        public AesCcm (byte[] key) => throw new PlatformNotSupportedException ();
```

Comment:
should we throw `NotImplementedException` instead?

Reply:
Correct me if I am wrong, but I believe our previous cryptography adds threw PNSE's.